README.md

```diff
@@ -56,13 +63,24 @@ cd microservices-demo
 
 3. **Create a GKE cluster.**
 
+- GKE standard mode:
```

Comment:
Does Autopilot have any missing features that impact the viability of the demo? If not, I'd suggest leading with Autopilot here.